### ISSUE TITLE
I think I broke the server

### ISSUE THREAD

**Issue by Orbitus2001** [RTRGitHub/RTRRedux#571]
Ever since last night, I have been having issues logging in to do anything on the server.  I log in and try to do anything and I can see my machines, but all of the Mekanism power conduits, fluid pipes, and gas pipes are segmented.  Meaning none of them are connected.  Also the new tank I put in for my power source is empty or at least not rendering anything in it.  I relog to see if that fixes it and all it does is either crash the server or causes the server to restart.  This is 2 nights now that it has been doing this.  And if I don't relog, the server eventually disconnects and kicks everyone else out.  

I also can't move from everything that is loaded from my loaded chunks when I log in.  I tried to go to spawn with the spawn command and it did nothing and I just sat there til it finally just disconnected.

**Comment by dragonaslin** [comment]
pop on tommorow night and ill run through it with ya and if it crashes you can i plz have a pastebin crash

**Comment by Orbitus2001** [comment]
Sorry, yesterday was Thanksgiving for the US, wasn't able to be on.  Solid said it was my Dynamic Tank from Mekanism causing the issue and had Rae attempt to replace the ticking blocks that were causing it.  But all of the blocks they replaced weren't even the tank.  So I left it there.  The last time I tried to log in though, I crashed the server again.  So this time, I am going to make something else to take the liquid ethylene I am making for a fuel source.

I just crashed the server again, and attempting to find a crash log.  My Minecraft didn't crash so I don't know if I will have one.
